src/parser.ts

```diff
@@ -0,0 +1,122 @@
+import { utils } from "ethers";
+import { parseString, RowValidateCallback } from "@fast-csv/parse";
+import { TokenMap } from "./hooks/tokenList";
+import { Message } from "./contexts/MessageContextProvider";
+import BigNumber from "bignumber.js";
+/**
+ * Includes methods to parse, transform and validate csv content
+ */
+
+export interface Payment {
+  receiver: string;
+  amount: BigNumber;
+  tokenAddress: string;
+  decimals?: number;
+}
+
+export type CSVRow = {
```

Comment:
Much better than SnakePayment!

Reply:
Although I would expect the linter to complain about `NotCamelCase` or something.